### DISCUSSION TITLE
Option to display ongoing connection line following pointer in "click-connect" mode

### DISCUSSION THREAD

**Discussion by lsdch** [bcakmakoglu/vue-flow#1414] [Ideas]
Hi, thanks for making this great tool available to the community.
Would you consider exposing an option to display a connection line between the source handle and the mouse pointer when using the "click-connect" feature ? As far as I understand, ongoing connection lines are currently only displayed when dragging from a handle.

**Comment by bcakmakoglu** [comment]
I'll think about it but this feature, if it's going to be added, won't land for a while.
